controller_manager/include/controller_manager/controller_manager.hpp

```diff
@@ -82,6 +82,13 @@ class ControllerManager : public rclcpp::Node
     const std::string & node_namespace = "",
     const rclcpp::NodeOptions & options = get_cm_node_options());
 
+  CONTROLLER_MANAGER_PUBLIC
```

Comment:
I wanted to have it to avoid creating a separate node especially for ResourceManager and use the one of ControllerManager. 

I believe we should have a constructor in future with no ResourceManager to be able to have the node built and then use it within ResourceManager construction